showcase/src/app/demo-ui/pages/input-mask-directives/demo-input-mask-directives.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, Inject, OnInit } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+import { StarkTextMasks, StarkTextMaskConfig } from "@nationalbankbelgium/stark-ui";
+
+@Component({
+	selector: "showcase-demo-mask-directives",
+	styleUrls: ["./demo-input-mask-directives.component.scss"],
+	templateUrl: "./demo-input-mask-directives.component.html"
+})
+export class DemoInputMaskDirectivesComponent implements OnInit {
+	public creditCardMaskConfig: StarkTextMaskConfig;
+	public structuredMessageMaskConfig: StarkTextMaskConfig;
+	public customMaskConfig: StarkTextMaskConfig;
+
+	public constructor(@Inject(STARK_LOGGING_SERVICE) private logger: StarkLoggingService) {}
+
+	public ngOnInit(): void {
+		this.creditCardMaskConfig = {
```

Comment:
I saw this in other places also, but why do we wait to set these values until OnInit? Wouldn't it be clearer to set them at definition?

Reply:
True, in these cases it doesn't harm to put them in the constructor... only in those cases where we rely on the `@Input`s but this isn't the case... I'll move them to the constructor 👍